.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,59 @@
+## Summary of Bug or Feature
+
+<!--
+Summarize the issue you are experiencing (or the feature you want to see added). Tell us what you were trying to do and what happened instead. Remember, this is _not_ a place to ask questions. For that, go to http://discuss.appium.io!
+
+THIS REPO IS FOR ISSUES RELATED TO THE PACKAGES IN appium/appium-ios ONLY!  See https://github.com/appium/appium/issues for general Appium issues.
+-->
+
+## Package(s) Affected (Required)
+
+<!--
+Check the package(s) this issue involves, and fill in the version number.
+-->
+
+- [ ] `@appium/idb` - version: <!-- version -->
+- [ ] `@appium/ios-device` - version: <!-- version -->
+- [ ] `@appium/ios-simulator` - version: <!-- version  -->
+- [ ] `@appium/ios-test-app` - version: <!-- version -->
+- [ ] `@appium/remote-debugger` - version: <!-- version -->
+- [ ] `@appium/xcode` - version: <!-- version -->
+
+## Details (Optional - Good to Have)
+
+<!--
+If possible, describe the problem you have been experiencing in more detail, add notes, etc. If this is a feature request: what might be an alternative solution?  Is there a workaround?
+-->
+
+<!-- *** BELOW IS FOR BUGS ONLY; IF REQUESTING A FEATURE, PLEASE DELETE TEXT BELOW *** -->
+
+## Environment (Required)
+
+<!--
+Please append the requested information to each list item below:
+-->
+
+- Appium version (or git revision) that exhibits the issue:
+- Last Appium version that did not exhibit the issue (if applicable):
+- Desktop OS/version used to run Appium:
+- Node.js version (unless using Appium.app|exe):
+- npm or Yarn package manager version:
+- Mobile platform/version under test:
+- Real device or emulator/simulator:
+- Appium CLI or Appium.app|exe:
+
+## Link to Appium logs (Required)
+
+<!--
+Do _NOT_ paste your full Appium logs here, as it will make this issue very long and hard to read!
```

Comment:
so does one need to paste the full or not?

Reply:
No, it says to use a gist (this is from the same template in appium/appium)

Reply:
yeah, maybe the wording is confusing